SymPEP-0001.md

```diff
@@ -0,0 +1,241 @@
+- To document the above discussions and decisions for future reference.
+- To document the motivations for a change from the perspective of end users
+  who may be affected by it. Here "end users" means both users of the SymPy
+  library, as well as developers working on parts of SymPy itself which may be
+  affected by the change.
+
+Importantly, **a SymPEP is not documentation** for the proposed change. End
+user documentation should be included with the implementation of the feature
+in the corresponding SymPy documentation. This also means that other
+documentation should not cross-reference a SymPEP as if it were documentation
+for a feature. Even technical discussion of a feature should be documented
+separately from a SymPEP. The reason is that SymPEPs will necessarily include
+details that are irrelevant to the final implementation, such as discussions
+of alternate implementations which were rejected and discussions of
+implementation of the change.
+
+## SymPEP Workflow
+
+Discussions around a proposed change typically begin informally on the mailing
+list or SymPy issue/pull-request tracker. However, once it is decided that the
+formal SymPEP process is desired for a change, the discussion should move to
+the [SymPEP repository](https://github.com/sympy/SymPEPs) on GitHub.
+
+Not all significant changes or additions to SymPy require a SymPEP. If the
+change would benefit from extended discussion or needs a roadmap for
+implementation, a SymPEP should be considered. If unsure, consult with the
+community on the currently active discussion forum. Most all changes to and
+decisions regarding SymPy can be made through issues and pull requests, so it
+is best to start with one or the other. Examples of things that likely need a
+SymPEP are: major version changes (1.X.X -> 2.0), large breaks in backwards
+compatibility, adding new primary packages within SymPy, adding hard
+dependencies, introduction of development practices or tools all developers
+need to adopt, etc. Long term SymPy policies and goals should also be discussed
+and developed as SymPEPs. Statements of these policies and goals should be
+incorporated into the SymPy documentation, as determined in the associated
+SymPEP.
+
+Every SymPEP must have at least one champion. The champion(s) are persons who
+are responsible for writing the SymPEP, leading the discussions around it, and
+developing consensus around it.
+
+The author of a SymPEP should fork the repository and create a draft pull
+request with a new SymPEP document based on the [SymPEP
+template](SymPEP-template). The SymPEP document may be named `SymPEP-XXXX.md`
+until a number is assigned. Once the author completes all information in the
+template and would like it to be reviewed, one of the core SymPy developers should
+decide if the proposal is a legtimate proposal and then assign a number to the
+SymPEP, in which case `XXXX` should be replaced with the number with leading
```

Comment:
legitimate

There is no definition of what a "SymPy core developer" is. I'm not sure if we want to make that formal or not.

Reply:
As I understand it, it a person that has merge rights on the sympy repo and thus in that github team.